azure/Guidelines.md

```diff
@@ -0,0 +1,864 @@
+# Microsoft Azure REST API Guidelines
```

Comment:
Are websockets and negotiating their creation out of scope of this document, or indeed in scope for the larger standardization effort in general? Several services in DevDiv use these and i think guidance would be valuable.

Reply:
It's out of scope for this document, but we want to add a separate doc for this document.
 https://github.com/microsoft/api-guidelines/issues/272